src/components/radioButton/RadioButton.tsx

```diff
@@ -216,7 +181,9 @@ class RadioButton extends BaseComponent {
         {...others}
         style={undefined}
         onPress={this.onPress}
-        {...this.getAccessibilityProps()}
+        accessibilityStates={disabled ? ['disabled'] : undefined}
+        accessibilityRole={'button'} // 'radio', TODO: uncomment when switching to RN60
```

Comment:
Technically we could change this to 'radio', are users with the current version should all be using RN61?

Reply:
I prefer to separate non typescript related changes from this PR. 
The todo is enough for now

Reply:
ok